blog/admin.py

```diff
@@ -1,7 +1,58 @@
+from blog.models import Blog, Comment, UserVote, Profile
+
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class ProfileInline(admin.StackedInline):
+    model = Profile
+    can_delete = False
+    verbose_name_plural = 'Profile'
+    fk_name = 'user'
+
+
+class CustomUserAdmin(UserAdmin):
+    inlines = (ProfileInline, )
+    list_display = (
+        'get_profile_id', 'id', 'username', 'first_name', 'last_name', 'get_date_of_birth', 'email', 'get_gender',
+        'get_contact_number', 'get_country'
+    )
+    list_select_related = ('profile',)
+
+    def get_date_of_birth(self, instance):
+        return instance.profile.date_of_birth
```

Comment:
We don't need to add method for direct model relations, e.g. you should be able to see the date of birth by simply adding `profile__date_of_birth` in the `list_display`. 

Same for other field methods here.

Reply:
Cannot directly display the date of birth because Profile is a one-to-one relation.